src/components/Integrations/Table/IntegrationTest.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { useState } from 'react';
+import axios from 'axios';
+import { Button, Modal, TextInput } from '@patternfly/react-core';
+import { useNotification } from '../../../utils/AlertUtils';
+
+const IntegrationTestModal = ({ integrationUUID, isModalOpen, onClose }) => {
+  const [inputValue, setInputValue] = useState('');
+  const { addSuccessNotification, addWarningNotification } = useNotification();
+
+  const placeholderText =
+    'Congratulations! The integration you created on https://console.redhat.com was successfully tested!';
+
+  const failedTestText =
+    'Your integration test has unfortunately failed, please verify your integration information.';
+
+  const handleNotificationTest = async (notificationMessage: string) => {
+    const endpointURL = `/api/integrations/v1/endpoints/${integrationUUID}/test`;
+
+    const body = { message: notificationMessage };
+
+    try {
+      const response = await axios.post(endpointURL, body);
+      response?.status === 204
+        ? addSuccessNotification('Integration Test', notificationMessage)
+        : addWarningNotification(
+            'Failed Test',
+            `Error through server response: ${response.status}`
+          );
+      onClose();
+    } catch (error) {
+      console.error('\nError sending test notification:', error);
+
+      const responseString = `${failedTestText} ${error}`;
+      addWarningNotification('Failed Test', responseString);
+      onClose();
+    }
+  };
+
+  return (
+    <Modal
```

Comment:
I think the modal should be 50 % or `medium` or something like that. It feels a bit odd to span across entire screen for just one input. PF has option to change the size, so we should utilize that.